source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -166,6 +166,14 @@ def output(self, key, obj):
     "address": AddressField()
 }
 
+geocode_poi = {
+    "embedded_type": Lit("poi"),
+    "quality": Lit("0"),
+    "id": fields.String(attribute='properties.geocoding.id'),
+    "name": fields.String(attribute='properties.geocoding.label'),
+    "poi": AddressField()
```

Comment:
You can't use AddressField(), will have to create PoiField, that will include:
```json
{
"address": {...},
"administrative_regions": [ ... ],
"coord": {...},
"id": "poi:7050",
"label": "Tour Eiffel (Paris)",
"name": "Tour Eiffel",
"poi_type": {...},
"properties": {}
}
```
Part of it is shared, though